diff --git a/node_modules/flowbite/dist/flowbite.js b/node_modules/flowbite/dist/flowbite.js
index ca66785..176758c 100644
--- a/node_modules/flowbite/dist/flowbite.js
+++ b/node_modules/flowbite/dist/flowbite.js
@@ -189,11 +189,11 @@ function initAccordion() {
 }
 
 if (document.readyState !== 'loading') {
-  // DOMContentLoaded event were already fired. Perform explicit initialization now
+  // turbo:load event were already fired. Perform explicit initialization now
   initAccordion();
 } else {
-  // DOMContentLoaded event not yet fired, attach initialization process to it
-  document.addEventListener('DOMContentLoaded', initAccordion);
+  // turbo:load event not yet fired, attach initialization process to it
+  document.addEventListener('turbo:load', initAccordion);
 }
 
 /* harmony default export */ const accordion = (Accordion);
@@ -302,11 +302,11 @@ function initCollapse() {
 }
 
 if (document.readyState !== 'loading') {
-  // DOMContentLoaded event were already fired. Perform explicit initialization now
+  // turbo:load event were already fired. Perform explicit initialization now
   initCollapse();
 } else {
-  // DOMContentLoaded event not yet fired, attach initialization process to it
-  document.addEventListener('DOMContentLoaded', initCollapse);
+  // turbo:load event not yet fired, attach initialization process to it
+  document.addEventListener('turbo:load', initCollapse);
 }
 
 /* harmony default export */ const collapse = (Collapse);
@@ -620,11 +620,11 @@ function initCarousel() {
 }
 
 if (document.readyState !== 'loading') {
-  // DOMContentLoaded event were already fired. Perform explicit initialization now
+  // turbo:load event were already fired. Perform explicit initialization now
   initCarousel();
 } else {
-  // DOMContentLoaded event not yet fired, attach initialization process to it
-  document.addEventListener('DOMContentLoaded', initCarousel);
+  // turbo:load event not yet fired, attach initialization process to it
+  document.addEventListener('turbo:load', initCarousel);
 }
 
 /* harmony default export */ const carousel = (Carousel);
@@ -704,11 +704,11 @@ function initDismiss() {
 }
 
 if (document.readyState !== 'loading') {
-  // DOMContentLoaded event were already fired. Perform explicit initialization now
+  // turbo:load event were already fired. Perform explicit initialization now
   initDismiss();
 } else {
-  // DOMContentLoaded event not yet fired, attach initialization process to it
-  document.addEventListener('DOMContentLoaded', initDismiss);
+  // turbo:load event not yet fired, attach initialization process to it
+  document.addEventListener('turbo:load', initDismiss);
 }
 
 /* harmony default export */ const dismiss = (Dismiss);
@@ -2854,11 +2854,11 @@ function initDropdown() {
 }
 
 if (document.readyState !== 'loading') {
-  // DOMContentLoaded event were already fired. Perform explicit initialization now
+  // turbo:load event were already fired. Perform explicit initialization now
   initDropdown();
 } else {
-  // DOMContentLoaded event not yet fired, attach initialization process to it
-  document.addEventListener('DOMContentLoaded', initDropdown);
+  // turbo:load event not yet fired, attach initialization process to it
+  document.addEventListener('turbo:load', initDropdown);
 }
 
 /* harmony default export */ const dropdown = (Dropdown);
@@ -3085,11 +3085,11 @@ function initModal() {
 }
 
 if (document.readyState !== 'loading') {
-  // DOMContentLoaded event were already fired. Perform explicit initialization now
+  // turbo:load event were already fired. Perform explicit initialization now
   initModal();
 } else {
-  // DOMContentLoaded event not yet fired, attach initialization process to it
-  document.addEventListener('DOMContentLoaded', initModal);
+  // turbo:load event not yet fired, attach initialization process to it
+  document.addEventListener('turbo:load', initModal);
 }
 
 /* harmony default export */ const modal = (Modal);
@@ -3421,11 +3421,11 @@ function initDrawer() {
 }
 
 if (document.readyState !== 'loading') {
-  // DOMContentLoaded event were already fired. Perform explicit initialization now
+  // turbo:load event were already fired. Perform explicit initialization now
   initDrawer();
 } else {
-  // DOMContentLoaded event not yet fired, attach initialization process to it
-  document.addEventListener('DOMContentLoaded', initDrawer);
+  // turbo:load event not yet fired, attach initialization process to it
+  document.addEventListener('turbo:load', initDrawer);
 }
 
 /* harmony default export */ const drawer = (Drawer);
@@ -3585,11 +3585,11 @@ function initTabs() {
 }
 
 if (document.readyState !== 'loading') {
-  // DOMContentLoaded event were already fired. Perform explicit initialization now
+  // turbo:load event were already fired. Perform explicit initialization now
   initTabs();
 } else {
-  // DOMContentLoaded event not yet fired, attach initialization process to it
-  document.addEventListener('DOMContentLoaded', initTabs);
+  // turbo:load event not yet fired, attach initialization process to it
+  document.addEventListener('turbo:load', initTabs);
 }
 
 /* harmony default export */ const tabs = (Tabs);
@@ -3761,11 +3761,11 @@ function initTooltip() {
 }
 
 if (document.readyState !== 'loading') {
-  // DOMContentLoaded event were already fired. Perform explicit initialization now
+  // turbo:load event were already fired. Perform explicit initialization now
   initTooltip();
 } else {
-  // DOMContentLoaded event not yet fired, attach initialization process to it
-  document.addEventListener('DOMContentLoaded', initTooltip);
+  // turbo:load event not yet fired, attach initialization process to it
+  document.addEventListener('turbo:load', initTooltip);
 }
 
 /* harmony default export */ const tooltip = (Tooltip);
